Substitute synced_files references with files_to_sync (#391)
packit/packit.dev: content/docs/CLI/propose-downstream.md

  upstream release tarball and uploads it to the lookaside cache. [This is
  required by the Fedora Packaging
  Guidelines](https://fedoraproject.org/wiki/Packaging:SourceURL#Referencing_Source).

>   Then it takes the spec file from the upstream repo, copies it downstream (with
>   also all the files set in the `files_to_sync`) and creates the downstream PR.

Reviewer: ```suggestion
Then it copies the files listed in `files_to_sync` (which by default includes the spec file) from the upstream repo to downstream and creates the downstream PR.
```
